dts/bindings/i2c/xlnx,xps-iic.yaml

```diff
@@ -0,0 +1,13 @@
+
+description: Xilinx PS IIC Bus Interface
+
+compatible: "xlnx,xps-iic"
```

Comment:
The I2C IPs present in the AMD Xilinx MPSoC are from Cadence, not from Xilinx.
This should be a Cadence driver matching `compatible = "cdns,i2c-r1p14"`